### PR TITLE
merge fix by nishadi for search by tag

### PR DESCRIPTION
**Pull request by danushkaf**

### CODE REVIEW

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java [hunk 1]
```diff
@@ -1971,4 +1968,66 @@ public void whiteListMaxAppCount(Connection dbConnection, int tenantId, int maxA
...
+    public List<Application> getTaggedApplicationsList(Connection dbConnection, int tenantId) throws AppCloudException {
+        PreparedStatement preparedStatement = null;
+        List<Application> taggedApplicationsList = new ArrayList<>();
+        Application application;
+        ResultSet resultSet = null;
+        try {
+            preparedStatement = dbConnection.prepareStatement(SQLQueryConstants.GET_ALL_APPLICATIONS_LIST_WITH_TAG);
+            preparedStatement.setInt(1, tenantId);
+            resultSet = preparedStatement.executeQuery();
+            Boolean applicationAddedtoList;
```
Comment (hisamith) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Why Boolean object? :open_mouth: :open_mouth:

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java [hunk 2]
```diff
@@ -1971,4 +1968,66 @@ public void whiteListMaxAppCount(Connection dbConnection, int tenantId, int maxA
...
+            while (resultSet.next()) {
+                Tag tag;
+                applicationAddedtoList = false;
+                //Iterating the existing tagged application list to check whether the application is already added into the list
+                for (Application applicationTemp : taggedApplicationsList) {
+                    String hashId = resultSet.getString(SQLQueryConstants.HASH_ID);
+                    System.out.println(hashId);
```
Comment (hisamith) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Sys out?? :P

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java (hunk 1 above)
Comment (danushkaf) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Fixed in https://github.com/wso2/app-cloud/pull/474

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java (hunk 2 above)
Comment (danushkaf) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Removed in https://github.com/wso2/app-cloud/pull/474

**Comment by nishadi** [comment]
Thank you for merging

**Pull request merged**